### PR TITLE
Fix SingIn and InputEmail and Add dashboard

### PR DESCRIPTION
**Pull request by TamaraKovalchuk**
Added last changes, added dashboard with scss file. Please check my SingIn and InputError components. And I also added dashboard for user, so check this too:) please.

### CODE REVIEW

File: src/modules/sign-in/SignIn.jsx [hunk 1]
```diff
@@ -63,10 +77,50 @@ export class SignIn extends Component {
         </Grid>
       )
     }
+  };
+
+  onChange = (e) => {
+   const name = e.target.name;
+   const value = e.target.value;
```
Comment (okosyk) [inline src/modules/sign-in/SignIn.jsx]: useless consts just set state directly from target

File: src/modules/sign-in/SignIn.jsx [hunk 2]
```diff
@@ -63,10 +77,50 @@ export class SignIn extends Component {
...
+   this.setState({[name]: value});
+  }
+
+  onBlur = (e) => {
+    const name = e.target.name;
```
Comment (okosyk) [inline src/modules/sign-in/SignIn.jsx]: same here

File: src/modules/sign-in/_index.scss [hunk 3]
```diff
@@ -31,4 +31,4 @@
     color: #f34b4b;
     height: 20px;
   }
-}
+}
```
Comment (okosyk) [inline src/modules/sign-in/_index.scss]: no empty line

File: src/modules/user-page/UserPage.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import UserDashboard from './dashboard/UserDashboard';
+
+class UserPage extends React.Component {
+    render() {
+    return (
+        <UserDashboard owner={'userId'}/>
+        );
+    }
+}
+export default UserPage;
```
Comment (okosyk) [inline src/modules/user-page/UserPage.jsx]: no empty line

File: src/router.jsx [hunk 5]
```diff
@@ -5,10 +5,12 @@ import SignIn from './modules/sign-in/SignIn';
 import SignUp from './modules/sign-up/SignUp';
 import Feed from './modules/feed/Feed';
 import EmailVerify from './modules/email-verify/EmailVerify';
+import UserPage from './modules/user-page/UserPage';
 
 const Routes = (
   <BrowserRouter basename="/">
     <Switch>
+      <Route exact path='/user-page' component={UserPage} />
```
Comment (okosyk) [inline src/router.jsx]: strange name for a route. 'profile' would be better

File: src/shared/components/_index.scss [hunk 6]
```diff
@@ -0,0 +1,2 @@
+@import './input-email/index';
+@import './form-error/index';
```
Comment (okosyk) [inline src/shared/components/_index.scss]: no empty line

File: src/shared/components/form-error/FormError.jsx [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const FormError = ({ formErrors }) => {
+  return <div className = "form-error">{formErrors}</div>
+}
```
Comment (okosyk) [inline src/shared/components/form-error/FormError.jsx]: no empty line

File: src/shared/components/input-email/InputEmail.jsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+  validate = (e) => {
+    const value = e.target.value;
+    let emailValid;
+    emailValid = value.length !== 0 && value.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i);
+    this.props.onValidate(emailValid && value.length!==0, value.length)
+  }
+
+  
+};
```
Comment (okosyk) [inline src/shared/components/input-email/InputEmail.jsx]: no empty line

File: src/shared/components/input-email/InputEmail.jsx [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+    return (
+      <FormControl margin={"normal"} fullWidth>
+        <InputLabel htmlFor="inp-email">Email</InputLabel>
+        <Input
+          id="inp-email"
+          name="email"
+          type={"email"}
+          onBlur={this.validate}
+          onChange={this.props.onChange}
+          required
```
Comment (okosyk) [inline src/shared/components/input-email/InputEmail.jsx]: put required at very top

 <Input required
    atrrs...

File: src/modules/sign-in/SignIn.jsx (hunk 1 above)
Comment (TamaraKovalchuk) [reply src/modules/sign-in/SignIn.jsx]: done
onChange = (e) => {
   this.setState({[e.target.name]: e.target.value});
  }

File: src/modules/sign-in/SignIn.jsx (hunk 2 above)
Comment (TamaraKovalchuk) [reply src/modules/sign-in/SignIn.jsx]: done
onBlur = (e) => {
    this.validateField(e.target.name);
    this.validateButtonState();
   }

File: src/modules/sign-in/_index.scss (hunk 3 above)
Comment (TamaraKovalchuk) [reply src/modules/sign-in/_index.scss]: done

File: src/modules/user-page/UserPage.jsx (hunk 4 above)
Comment (TamaraKovalchuk) [reply src/modules/user-page/UserPage.jsx]: done

File: src/router.jsx (hunk 5 above)
Comment (TamaraKovalchuk) [reply src/router.jsx]: <Route exact path='/profile' component={UserPage} />

File: src/shared/components/_index.scss (hunk 6 above)
Comment (TamaraKovalchuk) [reply src/shared/components/_index.scss]: done

File: src/shared/components/form-error/FormError.jsx (hunk 7 above)
Comment (TamaraKovalchuk) [reply src/shared/components/form-error/FormError.jsx]: done

File: src/shared/components/input-email/InputEmail.jsx (hunk 9 above)
Comment (TamaraKovalchuk) [reply src/shared/components/input-email/InputEmail.jsx]: done

File: src/shared/components/input-email/InputEmail.jsx (hunk 8 above)
Comment (TamaraKovalchuk) [reply src/shared/components/input-email/InputEmail.jsx]: done

File: src/router.jsx (hunk 5 above)
Comment (TamaraKovalchuk) [reply src/router.jsx]: done

**Pull request merged**